concent_api/utils/helpers.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+import time
+
+import calendar
+
+from django.utils import timezone
+
+
+def get_current_timestamp() -> int:
+    """ Returns current timestamp as int. """
```

Comment:
Please add more detail here:
- The timestamp is guaranteed to be in UTC.
    - Maybe it would even be a good idea to add `utc` in the name? I'm not sure but that would make it very clear. What do you think?
- The timestamp is not OS-dependent (uses the UNIX epoch).
    - I'm not 100% sure that it's true but this is what is supposed to be returned so if it's not true it's a bug.

Reply:
:+1: 
```
The timestamp is not OS-dependent (uses the UNIX epoch).
I'm not 100% sure that it's true but this is what is supposed to be returned so if it's not true it's a bug.
```
It uses time.time under the hood, so it will work correctly on OSes that handle epoch correctly.